de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java

```diff
@@ -64,6 +64,11 @@
 
 public class HULoader
 {
+	public static final HULoader of(final IAllocationSource source, final IAllocationDestination destination)
```

Comment:
good, but i propose to deprecate the public constructor (or make it private), to make sure that devs know the *one* way of getting a HULoader instance